kiwi/config/functions.sh

```diff
@@ -263,7 +263,7 @@ function baseSetupUserPermissions {
         fi
         if [[ ! "${shell}" =~ nologin|true|false ]];then
             group=$(grep "${group}" /etc/group | cut -f1 -d:)
-            chown -c -R "${usern}":"${group}" "${dir}"/*
+            chown -c -R "${usern}":"${group}" "${dir}"
```

Comment:
For me makes sense, but if we decided so we need to update the docs:
https://github.com/OSInside/kiwi/blob/master/doc/source/working_with_kiwi/shell_scripts.rst

Reply:
> Search all home directories of all users listed in :file:`/etc/passwd` and change the ownership of all files to belong to the correct user and group

I think this is more a philosophical question. Does /home/X belong to user X or /home/X/* ? My installation of SUSE Leap 15.0 makes ownership of /home/X to 755. In practice it belongs to user X and the script changes the ownership towards the user...

Reply:
When the user is created, useradd takes care of fixing the ownership and permission (755) of the directory, and copy the data from /etc/skel with the correct ownership + permissions.

The issue about the documentation is a detail, as is referring about changing the content and not the /home/xxx directory. For me ${dir} or ${dir}/* makes sense, but the doc is referring to the second instance, not the first one. So if you like to change the behavior, you need to update the documentation.

I personally do not see any advantage / disadvantage on any of both approach.

Reply:
I hope I understood it correctly: It is referring only to the **content** of /home/xxx only and should also refer to the homedirectory itself? See change

Reply:
Seems like there is an ongoing discussion. Labeled the PR as such

@aplanas could you answer the question ? Thanks

Reply:
iirc the doc and code change are now well aligned

@davidcassany can you have a short look too. For me the change looks good